node_app/models/Immunization.js

```diff
@@ -0,0 +1,146 @@
+/*
+Immunization {
+  vaccine: string
+  manufacturer: string "Organization/id"
+  lotNumber: string
+  expiration: string "YYYY, YYYY-MM, or YYYY-MM-DD"
+  patient: string "Patient/id"
+  encounter: string "Encounter/id"
+  status: enum (completed, entered-in-error, not-done)
+  statusReason: enum (IMMUNE, MEDPREC, OSTOCK, PATOBJ) i.e. immunity, medical precaution, product out of stock, patient objection
+  occurrence: string "YYYY, YYYY-MM, YYYY-MM-DD or YYYY-MM-DDThh:mm:ss+zz:zz"
+  primarySource: boolean
+  location: string "Location/id"
+  site: enum (LA, RA) i.e. left arm, right arm
+  route: enum (IDINJ, IM, NASINHLC, IVINJ, PO, SQ, TRNSDERM) i.e. , inj intramuscular, inhalation nasal, inj intravenous, swallow, inj subcutaneous, transdermal
+  doseQuantity: decimal
+  doseUnit: string
+  performer: [string] "{Practitioner/PractitionerRole/Organization}/id"
+  education: [string] "url"
```

Comment:
There are other education attributes we may need to capture, but I am not sure yet (like publication date, presentation date).  If you'd prefer to go ahead and add them now, that's fine.  Otherwise, I'll also approve this as is.

Reply:
This needs further discussion